webpack/assets/javascripts/react_app/components/toasts/Toast.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {Button} from 'react-bootstrap';
+
+const ToastIcons = {
+  success: {icon: 'ok', className: 'success'},
+  notice: {icon: 'ok', className: 'success'},
+  danger: {icon: 'error-circle-o', className: 'danger'},
+  error: {icon: 'error-circle-o', className: 'danger'},
+  warning: {icon: 'warning-triangle-o', className: 'warning'},
+  'default': {icon: 'info', className: 'info'}
+};
+
+class Toast extends React.Component {
+  constructor(props) {
+    super(props);
+    this.type = (props.type || 'success');
+  }
+  icon() {
+    return 'pficon pficon-' + (ToastIcons[this.type].icon || ToastIcons.default.icon);
+  }
+  closeBtn() {
+    return (
+      <Button className="close" data-dismiss="alert" aria-hidden="true">
+        <span className="pficon pficon-close"></span>
+      </Button>
+    );
```

Comment:
Why not use html button tag here as in patternfly reference markup. What do we gain from using the react-bootstrap component?